wp-content/themes/missilethreat/search.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/* 
+Template Name: Search Page Custom 
+*/
+
+get_header();
+?>
+
+<main id="site-content" role="main">
+
+<?php
+
+global $wp_query;
+
+if ( $wp_query->found_posts ) {
```

Comment:
You should be able to use `if ( have_posts() )` here instead of calling the `global $wp_query`;

Reply:
Is this just for the if statement?  Asking because the `global $wp_query` is being used for the number of search results and the internet seems to think that's the way to do it.